src/components/NotesMindMap/NotesMindMapActions.js

```diff
@@ -21,6 +21,7 @@ export const CHANGE_PARENT_REQUEST_SUCCESS_ACTION =
   'CHANGE_PARENT_REQUEST_SUCCESS_ACTION';
 export const CHANGE_PARENT_REQUEST_FAIL_ACTION =
   'CHANGE_PARENT_REQUEST_FAIL_ACTION';
+export const SAVE_SEARCH_RESULTS = 'SAVE_SEARCH_RESULTS';
```

Comment:
We're following the Redux styleguide, in particular the rule to [model action as events and not setters](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters).
So this action should be `SEARCH_REQUEST_SUCCESS`, instead of `SAVE_SEARCH_RESULTS`.

Reply:
Changed, thanks for the observation.